app/assets/javascripts/comments.js

```diff
@@ -3,4 +3,12 @@ $(document).ready(function() {
     e.preventDefault();
     $(e.target).parents("ul").next('.comment-reply').toggleClass('hidden');
   });
-});
\ No newline at end of file
+});
+
+
+$(document).ready(function() {
+    $( '[data-detail-id^=comments-]' ).each(function(index) {
+      var comments = $( '#comments-'+index ).find(".comment").length;
```

Comment:
This variable should probably communicate the idea of "number of comments" instead of just "comments"